src/Controls/src/Core/PointerEventArgs.cs

```diff
@@ -11,15 +11,28 @@ public class PointerEventArgs : EventArgs
 
 		Func<IElement?, Point?>? _getPosition;
 
+		/// <summary>
+		/// Platform-specific arguments associated with the PointerEventArgs
```

Comment:
Since this is a property, you should specify if we get it or set it. 
```suggestion
		/// Gets the platform-specific arguments associated with the <see cref="PointerEventArgs"/>.
```
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property